packages/oidc/package.json

```diff
@@ -14,7 +14,7 @@
       "require": "./dist/index.js"
     }
   },
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Are we supposed to increment this, or is the changeset process supposed to increment this?